requirements.txt

```diff
@@ -16,3 +16,4 @@ ruamel.yaml==0.17.21
 PyGithub==1.57
 black==22.12.0
 flake8==6.0.0
+-e git+https://git@github.com/codeEmpress1/canonicalwebteam.store-base.git@2cfb9162a2521e6aa21461a76bb48478bf404d18#egg=canonicalwebteam_store_base
```

Comment:
does the image build if you remove the `-e` so it's not editable?

Reply:
I removed the -e and it still doesn't build.  It couldn't install store base from git